packages/map.vm/tools/chocolateyinstall.ps1

```diff
@@ -25,6 +25,16 @@ try {
   $shortcut = Join-Path $shortcutDir "$toolName.lnk"
   Install-ChocolateyShortcut -shortcutFilePath $shortcut -targetPath $toolDir
   VM-Assert-Path $shortcut
+
+  $cmdHereRegistryPath1 = "HKLM:\SOFTWARE\Classes\Directory\background\shell\ctx_cmd"
```

Comment:
I think I would move the catch before this code and add `$ErrorActionPreference = 'Continue'` to not fail the package if modifying the registry fails (then you can also simplify the code removing the Test-Path).